packages/debugger/app/components/frame-debugger.tsx

```diff
@@ -599,6 +611,25 @@ export const FrameDebugger = React.forwardRef<
                     FrameImage={FrameImageNext}
                     allowPartialFrame={true}
                   />
+                  {currentFrameStackItem?.status === "done" && (
+                    <WithTooltip tooltip="Open in Playground">
+                      <Link
+                        href={{
+                          pathname: "/playground",
+                          query: {
+                            frame: JSON.stringify(
+                              frameResultToPlaygroundFrame(
+                                currentFrameStackItem.frameResult
+                              )
+                            ),
+                          },
+                        }}
+                        className="absolute top-0 right-0 p-1 m-2 border border-slate-50 rounded-full z-10 shadow"
+                      >
+                        <TerminalIcon size={24} />
+                      </Link>
+                    </WithTooltip>
+                  )}
```

Comment:
I am worried about the positioning of this button - perhaps below the frame next to the frame URL, or in the action bar to the left or in the debugger could be better options for placements?

Reply:
Will do

Reply:
I moved it to actions on the left.